.github/workflows/node-pr.yml

```diff
@@ -43,6 +51,14 @@ on:
         description: "If the format step should skipped"
         default: false
         type: boolean
+      skip-test-storybook:
+        description: "If the test-storybook step should skipped"
+        default: false
```

Comment:
I feel like this should skip by default. Just because this node-pr workflow could be used in non FE projects

Reply:
ah yes, you are correct